products/template/sql/_intermediate_models.yml

```diff
@@ -0,0 +1,53 @@
+version: 2
+
+models:
+  - name: libraries
+    description: "All libraries in NYC"
+    columns:
+      - name: library_name
+        tests:
+          - not_null
+      - name: borough
+      - name: wkb_geometry
+    tests:
+      - unique:
+          name: libraries_unique_compound_key
+          column_name: "(library_name || '-' || borough)"
```

Comment:
Just chatted in person, but this syntax is awkward (and as it turns out, doesn't quite work as expected) compared with dbt_utils.unique_combination_of_columns

Reply:
will drop use of the built-in unique test for compound keys